src/test/run-make/process-termination/Makefile

```diff
@@ -0,0 +1,20 @@
+-include ../../run-make-fulldeps/tools.mk
+
+# ignore-wasm
+
+# For cargo setting
+RUSTC := $(RUSTC_ORIGINAL)
+LD_LIBRARY_PATH := $(HOST_RPATH_DIR)
+
+# We need to be outside of 'src' dir in order to run cargo
+WORK_DIR := $(TMPDIR)
+HERE := $(shell pwd)
+
+CRATE := accept
+
+all:
+	env
+	cp -a $(HERE)/$(CRATE) $(WORK_DIR)/$(CRATE)
+	cd $(WORK_DIR)/$(CRATE) && \
+	$(CARGO) build --target $(TARGET) --verbose && \
+	timeout 10s $(CARGO) run --target $(TARGET)
```

Comment:
So why exactly does this need to be a run-make test -- is it this `timeout 10s` call? i.e., why can't we just make this a normal `run-pass` test?